tests/test_csc.py

```diff
@@ -0,0 +1,154 @@
+# This file is part of dm_OCPS.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import asynctest
```

Comment:
This is a 3rd party library, and one that is no longer supported.

You are welcome to use it here (but if you do, please list it after the standard library imports).

However, please consider using unittest.IsolatedAsyncioTestCase instead. I have not updated my packages yet because a test will hang in Python 3.8.6 if the test exits with asyncio.CancelledError (because it is no longer an Exception, despite the name, which broke a lot of code). But we'll all be at Python 3.8.8 soon enough. And exiting with asyncio.CancelledError only can happen if the test fails.